types/methods/template.d.ts

```diff
@@ -1,12 +1,14 @@
 import type {Result, SyncResult} from '../return/result.js';
 
-// Values allowed inside `...${...}...` template syntax
 type TemplateExpressionItem =
 	| string
 	| number
 	| Result
 	| SyncResult;
 
+/**
+Value allowed inside `${...}` when using the template string syntax.
```

Comment:
Looks like it is missing the * on this line.

Reply:
This is not required.